blyp/Views/OnboardingView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  OnboardingView.swift
+//  blyp
+//
+//  Created by Shakeel Khan on 5/1/20.
+//  Copyright © 2020 Team Sonar. All rights reserved.
+//
+
+import SwiftUI
+
+struct OnboardingView: View {
+    
+    //Place to add images if desired.
+    var subviews = [
+        UIHostingController(rootView: Subview(imageString: "meditating")),
+        UIHostingController(rootView: Subview(imageString: "skydiving")),
+        UIHostingController(rootView: Subview(imageString: "sitting"))
+    ]
+    
+    var titles = ["Create and share Blyps!", "Appoint a legacy contact", "Automatic posting"]
+    
+    var captions =  ["Blyps are digital memories of any media: photos, videos, or just text that you can share with your loved ones!", "Legacy contacts are specially appointed by you to look after your account after your passing", "Select a future date or after your passing and your Blyp will be posted for you at that time!"]
+    
+    @State var currentPageIndex = 0
```

Comment:
[State vars should be private](https://developer.apple.com/documentation/swiftui/state)